pom.xml

```diff
@@ -5,13 +5,13 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
```

Comment:
I think you are using tabs for indentation.  use spaces instead for tabs. you no need to change manually. Intellij can do that when you press tab.